packages/virtual-core/src/index.ts

```diff
@@ -391,7 +413,7 @@ export class Virtualizer<TScrollElement = unknown, TItemElement = unknown> {
     },
   )
 
-  private calculateRange = memo(
+  calculateRange = memo(
```

Comment:
I'm ecstatic that you are trying to solve this, because my team recently just ran head first into this exact issue. My main question with this approach, I think, is why can't we just have `measure` (which is already public) call `calculateRange` as part of its implementation. At the moment, it already doesn't seem to be doing what it semantically says it should be doing (from what I can tell{).

Reply:
Hi, i think we need to re-thing measure, now it's only clears itemMeasurementsCache and trigger update so most cases it will not measure anything. When we merge #405 with adding support observing elements with ResizeObserver i think we could drop it.